### PR TITLE
Add vietnamese translation for Advance - Erlang Interoperability

### PR DESCRIPTION
**Pull request by vnnoder**
@joneslee85 please help to review and improve. 
I will translate Advanced lessons to learn at the same time :smiley:

### CODE REVIEW

File: vi/lessons/advanced/erlang.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-- [Erlang Packages](#erlang-packages)
-- [Notable Differences](#notable-differences)
-  - [Atoms](#atoms)
-  - [Strings](#strings)
-  - [Variables](#variables)
+- [Thư viện chuẩn](#standard-library)
+- [Erlang Packages (thư viện bên thứ ba)](#erlang-packages)
+- [Những điểm khác biệt cần lưu ý](#notable-differences)
+  - [Atom](#atoms)
+  - [Chuỗi](#strings)
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: Please revert to `String`. We could always append explanation clause in the header.

File: vi/lessons/advanced/erlang.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+## Mục lục
 
-- [Standard Library](#standard-library)
-- [Erlang Packages](#erlang-packages)
-- [Notable Differences](#notable-differences)
-  - [Atoms](#atoms)
-  - [Strings](#strings)
-  - [Variables](#variables)
+- [Thư viện chuẩn](#standard-library)
+- [Erlang Packages (thư viện bên thứ ba)](#erlang-packages)
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: please remove `(thư viện bên thứ ba)` here to keep the header short and concise.

File: vi/lessons/advanced/erlang.md [hunk 3]
```diff
@@ -79,9 +79,9 @@ Erlang:
 example.
 ```
 
-### Strings
+### Chuỗi
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: s/Chuỗi/String (chuỗi)

File: vi/lessons/advanced/erlang.md [hunk 4]
```diff
@@ -79,9 +79,9 @@ Erlang:
...
 
-In Elixir when we talk about strings we mean UTF-8 encoded binaries.  In Erlang, strings still use double quotes but refer to char lists:
+Khi nói đến chuỗi trong Elixir ta nói đến chuỗi nhị phân được mã hoá theo UTF-8. Với Erlang, chuỗi vẫn sử dụng double quotes (dấu phẩy kép ") nhưng lại là các char list (danh sách dấu chữ):
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: s/chuỗi/string

File: vi/lessons/advanced/erlang.md [hunk 5]
```diff
@@ -107,7 +107,7 @@ false
 true
 ```
 
-It's important to note that many older Erlang libraries may not support binaries so we need to convert Elixir strings to char lists.  Thankfully this is easy to accomplish with the `to_char_list/1` function:
+Điều cần phải chú ý ở đây là nhiều thư viện cũ của Erlang không hỗ trợ chuỗi nhị phân, vì vậy ta phải biến đổi chuỗi Elixir thành char list (danh sách dấu chữ):
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: `danh sách dẫu chữ` - what do you think about `danh sách ký tự`?

File: vi/lessons/advanced/erlang.md [hunk 6]
```diff
@@ -1,28 +1,28 @@
...
 layout: page
-title: Erlang Interoperability
+title: Tương tác với Erlang
 category: advanced
 order: 1
 lang: vi
 ---
 
-One of the added benefits to building on top of the ErlangVM is the plethora of existing libraries available to us.  Interoperability allows us to leverage those libraries and the Erlang standard lib from our Elixir code.  In this lesson we'll look at how to access functionality in the standard lib along with third-party Erlang packages.
+Một trong những lợi ích có được khi xây dựng dựa trên ErlangVM (máy ảo Erlang) là sự phong phú của các thư viện có sẵn. Khả năng tương tác với Erlang cho phép ta sử dụng những thư viện này cũng như thư viện chuẩn của Erlang khi viết code bằng Elixir. Ở bài này, chúng ta sẽ xem xét việc truy cập những tính năng của thư viện chuẩn cũng như thư viện bên thứ ba của Erlang.
```
Comment (joneslee85) [inline vi/lessons/advanced/erlang.md]: I don't think we need to explain ErlangVM here.

File: vi/lessons/advanced/erlang.md (hunk 5 above)
Comment (vnnoder) [inline vi/lessons/advanced/erlang.md]: Much better

**Comment by vnnoder** [comment]
@joneslee85 added a commit based on your feedback

**Pull request merged**